### PR TITLE
feat: Add lesson 9

### PR DESCRIPTION
**Pull request by Hkataya**

### CODE REVIEW

File: lesson09.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+console.log(me.name)
+console.log(me.address.city)
+me.name = "Hasan"
+console.log(me.name)
+```
+
+* What's the result?
+
+```js
+Ownen
```
Comment (olof-nord) [inline lesson09.md:29]: typo: Owen (no n)

**Review by olof-nord — COMMENTED** [review]
DId a quick read-through and just noticed what I believe is a spelling error.

File: lesson09.md [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+me.introduce();
+console.log(me.address.city);
+console.log(me.address.country);
+```
+
+---
+
+### "this" in JavaScript
+
+In JavaScript, ```this```  is a special keyword that refers to the object that the current function is a method of.
```
Comment (haraldF) [inline lesson09.md:73]:
```suggestion
In JavaScript, `this`  is a special keyword that refers to the object that the current function is a method of.
```

File: lesson09.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+
+```html
+<button onclick="sayHello()">Click Me</button>
+```
+
+In this example, When a user clicks the button, it triggers the sayHello function
+
+```js
+function sayHello() {
+    alert('Hello, World!');
```
Comment (haraldF) [inline lesson09.md:178]:
```suggestion
    console.log('Hello, World!');
```

No one uses alert :)

File: lesson09.md [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+}
+```
+
+
+---
+
+### Changing the background color
```
Comment (haraldF) [inline lesson09.md:185]: We already did that exercise, I think you can safely remove this and the next two slides, which should give us more time to focus on `getElementById`

File: lesson09.md [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+```
+---
+
+### HTML and Javascript
+
+Javascript (Interactions): Javascript enables interactivity on a webpage. It allows you to respond to events like clicks, input, and more. JavaScript can change the content, style, and behavior of the webpage dynamically.
+
+```js
+function changeText() {
+  document.getElementById("greeting").textContent = "Hi!";
```
Comment (haraldF) [inline lesson09.md:132]: The order here is a bit odd - we're using `getElementById` here (also - chaining???) but we formally introduce it below. Maybe we can remove theses three slides or move them below the `getElementById` formal introduction?

Comment (Hkataya) [reply lesson09.md]: reordered the slides in a better way

**Pull request merged**